async/kafka/group/group.go

```diff
@@ -164,10 +164,14 @@ func (h handler) ConsumeClaim(sess sarama.ConsumerGroupSession, claim sarama.Con
 	ctx := sess.Context()
 	for msg := range claim.Messages() {
 		kafka.TopicPartitionOffsetDiffGaugeSet(h.consumer.group, msg.Topic, msg.Partition, claim.HighWaterMarkOffset(), msg.Offset)
+		kafka.MessageStatusCountInc("received", h.consumer.group, msg.Topic)
```

Comment:
some of the statuses can be created as consts and share from the kafka package

Reply:
I did this for all the MessageStatus labels, and only exported those who needed exporting.   
I left the `ACK` `NACK` labels as-is, it's straightforward this way.